nvOC_miner_update.sh

```diff
@@ -25,354 +134,828 @@ function restart-if-needed {
   fi
 }
 
+# Need dual source update for new miners with cuda 9 support
```

Comment:
Take the updated implementation for this function from my branch, use the hashes I asked you previously as third argument for this function in compile sections.